### PR TITLE
#249540 Add support for angular templateUrl & styleUrls

### PR DESCRIPTION
**Pull request by jordanilchev**
+ plus small css tweaks to custom Title field famous  "green poison border"

### CODE REVIEW

File: custom-fields/custom-field-readonly.component.ts [hunk 1]
```diff
@@ -2,14 +2,16 @@ import { Component } from "@angular/core";
...
-        `
-            .custom-input {
-                height: 100px;
-                border: 2px solid red;
-            }
-        `
-    ]
+    templateUrl: "./custom-field-readonly.component.html",
+    styles: [`
+        .custom-title-input {
```
Comment (belchevgb) [inline custom-fields/custom-field-readonly.component.ts]: Maybe this should be in .css file also, just like custom-field-write.component.css ?

File: sitefinity-videos/sitefinity-videos-toolbar-item-provider.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { Injectable, ClassProvider, Inject } from "@angular/core";
 import { ToolBarItem, ToolBarItemsProvider, TOOLBARITEMS_TOKEN, SelectorService, SelectorOptions, SELECTOR_SERVICE } from "progress-sitefinity-adminapp-sdk/app/api/v1";
 
-require("./sf-video-toolbar.css");
+require("!style-loader!css-loader!./sitefinity-videos-toolbar-item-provider.css");
```
Comment (belchevgb) [inline sitefinity-videos/sitefinity-videos-toolbar-item-provider.ts]: Maybe "!style-loader!css-loader!" should be as a constant and used in interpolated string, so the next time we change the loaders, we don't have to change these strings.

File: custom-fields/custom-field-readonly.component.ts (hunk 1 above)
Comment (jordanilchev) [reply custom-fields/custom-field-readonly.component.ts]: I left it this way to showcase both possibilities, but good point to add comment on both places

File: sitefinity-videos/sitefinity-videos-toolbar-item-provider.ts (hunk 2 above)
Comment (jordanilchev) [reply sitefinity-videos/sitefinity-videos-toolbar-item-provider.ts]: I prefer to leave it this way as this is the webpack specific scenario and users will find support and docs much easy directly via webpack docs (not the best, but still no need to abstract it and take support on us)

File: custom-fields/custom-field-readonly.component.ts (hunk 1 above)
Comment (jordanilchev) [reply custom-fields/custom-field-readonly.component.ts]: done

File: sitefinity-videos/sitefinity-videos-toolbar-item-provider.ts (hunk 2 above)
Comment (jordanilchev) [reply sitefinity-videos/sitefinity-videos-toolbar-item-provider.ts]: done

**Pull request merged**